### ISSUE TITLE
Closing a renderInPlace popover via yielded action fires onHidden twice

### ISSUE THREAD

**Issue by roberkules** [kaliber5/ember-bootstrap#750]
I'm closing a popover using the `po.close` yielded action, but when I do so it requires 2 clicks on the trigger element to open the popover again.

I quickly tried to reproduce this behavior on ember-twiddle but failed. still managed to see a weird behavior though and I wonder if that could be related:

https://ember-twiddle.com/e48db7c37c9055a016560bbc69dcca96?openFiles=templates.application.hbs%2C

when the popover is closed by clicking on the initial trigger element, it closes and the `onHidden` hook is called once. but when I close it using the `close` method, the `onHidden` hook is called twice

---

Output from `ember bootstrap:info`:
```
Npm packages:
ember-bootstrap: ~2.4.0 -> 2.4.0
ember-cli: ~3.2.0 -> 3.2.0
bootstrap: ~4.1.3 -> 4.1.3
bootstrap-sass: n/a
ember-cli-sass: ~8.0.1 -> 8.0.1
ember-cli-less: n/a
Bower packages:
bootstrap: n/a
bootstrap-sass: n/a
ember-bootstrap configuration:
bootstrapVersion: 4
importBootstrapFont: false
importBootstrapCSS: false
```

**Comment by simonihmig** [comment]
@roberkules thanks for reporting and the reproduction!

In your twiddle, the popover is rendered within the buttons DOM, probably because ember-bootstrap's generated div where all wormholed elements (modals, popovers, tooltips) are rendered to is not present. In this case it behaves as if `renderInPlace=true`. So being technically part of the button, when you click the close link in the popover (and not preventing the event to bubble up), the button effectively gets also clicked, thus causing the second `onHidden`. You can see that when holding the mouse on the click link, the button is also in a pressed state!

Though quite an edge case, this certainly should be fixed.

But not sure, as you said, it's the same problem you have seen in the first place? Is your popper rendered within its parent (e.g. a button)?

**Comment by simonihmig** [comment]
I renamed the issue, to track the specific case demonstrated in the twiddle. If you see that your actual problem is a different one, then please file a new issue!

**Comment by roberkules** [comment]
@simonihmig thanks for looking into it. as for my original issue, I could reproduce it adding the `ember-bootstrap-wormhole` element to the application template and opened a new ticket.

as for the issue with `onHidden` being called twice ... since I worked on the same twiddle for both issues, here is the [old version of the twiddle](https://ember-twiddle.com/e48db7c37c9055a016560bbc69dcca96/5172ea461dd6664b01b05b911b38eca4bca817f3?openFiles=controllers.application.js%2C)

thx, robert

**Comment by roberkules** [comment]
@simonihmig seems like this issue has been resolved, I upgraded to 2.7.x and it's calling onHidden only once as expected :)

**Comment by simonihmig** [comment]
@roberkules oh, that actually came unexpected. There were some changes regarding popovers, but not sure what could have caused this bug to "accidentally be fixed"!? 🤷‍♂️

But I was able to reproduce when updating ember-bootstrap in your twiddle! So closing this for now.
